geoserver_init/010_init-gs.js

```diff
@@ -109,20 +111,23 @@ async function createPostgisDatastore() {
 /**
  * Creates the station layer.
  */
-// async function createStationsLayer() {
-//   framedMediumLogging('Creating stations layer...');
-//
-//   const workspace = 'station_data';
-//   const dataStore = 'station_data';
-//   const stationLayerName = 'fv_stations';
-//   const srs = 'EPSG:3035';
-//
-//   const success = await grc.layers.publishFeatureType(workspace, dataStore, stationLayerName, stationLayerName, stationLayerName, srs);
-//
-//   if (success) {
-//     console.info('Successfully created stations layer ', stationLayerName);
-//   }
-// }
+async function createStationsLayer() {
+  framedMediumLogging('Creating stations layer...');
+
+  const workspace = 'station_data';
+  const dataStore = 'station_data';
+  const stationLayerName = 'fv_stations';
+  const stationLayerTitle = 'All Stations';
+  const srs = 'EPSG:3035';
+
+  const success = await grc.layers.publishFeatureType(workspace, dataStore, stationLayerName, stationLayerName, stationLayerTitle, srs);
+
+  if (success) {
+    console.info('Successfully created stations layer ', stationLayerName);
+  } else {
+    console.error('Failed creating stations layer', stationLayerName);
```

Comment:
whitespace is missing
```suggestion
    console.error('Failed creating stations layer ', stationLayerName);
```

Reply:
The whitespace is added by the log function. Therefore I removed the extra whitespace in the info log above by rebasing

Reply:
okay